packages/dashboard/src/components/Section/SectionPage.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { ExampleCard, IComponentDemoPageProps, ComponentPage, PropertiesTableSet } from '@uifabric/example-app-base';
+import { SectionBasicExample } from './examples/Section.Basic.Example';
+const SectionBasicExampleCode = require('!raw-loader!@uifabric/dashboard/src/components/Section/examples/Section.Basic.Example.tsx') as string;
+
+export class SectionPage extends React.Component<IComponentDemoPageProps, {}> {
+  public render(): JSX.Element {
+    return (
+      <ComponentPage
+        title="Section"
+        componentName="Section"
+        exampleCards={
+          <div>
+            <ExampleCard title="Section" isScrollable={true} isOptIn={true} code={SectionBasicExampleCode}>
+              <SectionBasicExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet
+            sources={[require<string>('!raw-loader!@uifabric/dashboard/src/components/Section/Section.types.ts')]}
+            // renderOnly={['IDashboardGridLayoutProps', 'DashboardGridBreakpointLayouts', 'IDashboardCardLayout']}
```

Comment:
Remove commented line?

Reply:
Done.

---
In reply to: [210771917](https://github.com/OfficeDev/office-ui-fabric-react/pull/5899#discussion_r210771917) [](ancestors = 210771917)